### PR TITLE
Adding longer descriptions

### PR DESCRIPTION
**Pull request by benarent**
Feedback from our SEO agency to add better descriptions

### CODE REVIEW

File: docs/pages/access-controls/guides/role-templates.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Teleport Role Templates
-description: Mapping SSO and Local Users Traits to Roles with Template.
+description: In this guide, learn how Teleport's role templates provide a way to describe these and other policies.
```
Comment (ptgott) [inline docs/pages/access-controls/guides/role-templates.mdx:3]: What does "these" refer to here? The policies in this guide? I'm not sure what "other" would mean in that case.

File: docs/pages/access-controls/reference.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Access Controls Reference Documentation
-description: Access Controls reference documentation for role options and properties.
+description: Learn how to use Teleport roles to manage role-based access controls (RBAC) in your Teleport cluster with this helpful guide.
```
Comment (ptgott) [inline docs/pages/access-controls/reference.mdx:3]:
```suggestion
description: Learn how to use Teleport roles to manage role-based access controls (RBAC) in your Teleport cluster with this detailed guide.
```

Leaving it to the reader to determine if it's helpful

File: docs/pages/management/operations/upgrading.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Upgrading a Teleport Cluster
-description: How to upgrade Teleport components
+description: Learn how to upgrade all of the components in your Teleport cluster with this helpful guide.
```
Comment (ptgott) [inline docs/pages/management/operations/upgrading.mdx:3]: Same re: "helpful guide"

File: docs/pages/reference/networking.mdx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Networking
-description: Teleport ports and networking requirements
+description: In this guide you'll learn how about Teleport ports and networking requirements.
```
Comment (ptgott) [inline docs/pages/reference/networking.mdx:3]: "learn how about" isn't a standard English usage, AFAIK

File: docs/pages/access-controls/guides/role-templates.mdx (hunk 1 above)
Comment (zmb3) [reply docs/pages/access-controls/guides/role-templates.mdx:3]: +1, I don't understand what this is saying

File: docs/pages/access-controls/reference.mdx (hunk 2 above)
Comment (zmb3) [reply docs/pages/access-controls/reference.mdx:3]: I don't see this as an improvement. Do we have unhelpful (or undetailed) guides too? What stops us from just putting "with this helpful guide" on every single guide? Should we?

I am not an expert in SEO, but to me this just looks like extra words, and our general guidance thus far in docs has been to be clear and concise and avoid adding extra fluff.

I'm curious what @klizhentas thinks about this.

Comment (benarent) [reply docs/pages/access-controls/reference.mdx:3]: The purpose behind these is to better control the meta descriptions, for example, what we currently have in `description: ` is ignored by search engines.  It instead picks snippets for us. These are mostly suggestions, I can rework them to remove the extra words. We ideally want meta descriptions that aren't concatenated. 

image
image

**Comment by benarent** [comment]
I'm going to close this issue and loop back with the team on better way to update these.

**Pull request closed without merging**